api/opencl_runtime_layer.asciidoc

```diff
@@ -12940,6 +12940,7 @@ Please refer to handle specific documentation for more details on transference r
 To export an external handle from a semaphore, call the function
 
 include::{generated}/api/protos/clGetSemaphoreHandleForTypeKHR.txt[]
+include::{generated}/api/version-notes/clGetSemaphoreHandleForTypeKHR.asciidoc[]
```

Comment:
I'm wondering whether we should include the version notes in the generated proto snippet. It seems we mostly (always?) include them right after the proto snippet.

Reply:
I honestly can't remember if there was a reason I put them in a separate file, rather than adding them to the proto.